player/csndfile.c

```diff
@@ -135,7 +137,7 @@ void csf_init_instrument(song_instrument_t *ins, int samp)
 
 song_instrument_t *csf_allocate_instrument(void)
 {
-	song_instrument_t *ins = mem_calloc(1, sizeof(song_instrument_t));
+	song_instrument_t *ins = mem_alloc(sizeof(song_instrument_t));
 	csf_init_instrument(ins, 0);
```

Comment:
Correct me if I'm wrong, but aren't there a bunch of `song_instrument_t` fields that aren't initialized by `csf_init_instrument`?

Reply:
This commit moves the memset into `csf_init_instrument()`.
https://github.com/schismtracker/schismtracker/pull/139/files#diff-31edc9eb6dab37d1e6a5f1d7feec5b2cR123

Requiring one to zero the reused instrument prior to initialization with a memset seemed janky to me...

Reply:
Ah, got it. I was looking at the files in master for reference.